source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,106 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
+
+1. Log into your Asana account.
+2. From the Asana Board Menu, select **Export/Print > JSON** to create an archive file (which you'll use in Boards).
+3. Save the file locally, e.g. to ``asana.json``.
+4. In your local copy of Boards, open a terminal window.
+5. Navigate to ``focalboard/webapp``.
+6. Run ``npm install``.
+7. Change directory to ``focalboard/import/asana``.
+8. Run ``npm install``.
+9. Run ``npx ts-node importAsana.ts -i <asana.json> -o archive.focalboard``.
+10. Open Boards and select **Settings > Import archive** and select ``archive.focalboard``.
+11. Select **Upload**.
+12. Return to your board and confirm that your Asana data is now displaying.
```

Comment:
Is this the expected conclusion? What happens if they don't see their Asana data?

Reply:
Yes. The board will be added to the sidebar. If they don't see their Asana data, the error should be displayed but they should also check log files for errors.